src/lib/Components/Artist/ArtistShows/Metadata.tsx

```diff
@@ -19,7 +19,7 @@ class Metadata extends React.Component<Props> {
     return (
       <View style={styles.container}>
         {!!partnerName && <Text style={styles.sansSerifText}>{partnerName.toUpperCase()}</Text>}
-        <Text style={styles.sansSerifText}>{this.showTypeString()}</Text>
+        {!!this.showTypeString() && <Text style={styles.sansSerifText}>{this.showTypeString()}</Text>}
```

Comment:
A minor piece of feedback, but instead of calling `showTypeString()` twice, we could store it in a local variable above the `return` statement. Purely a stylistic change, but feels nicer (to me, anyway).

Reply:
👍